package.json

```diff
@@ -45,15 +45,18 @@
     "watch:labextension": "jupyter labextension watch ."
   },
   "dependencies": {
-    "@jupyterlab/application": "^3.1.0"
+    "@jupyterlab/application": "^3.1.0",
+    "@jupyterlab/launcher": "^3.2.7",
+    "@jupyterlab/mainmenu": "^3.2.6",
+    "tui-image-editor": "^3.15.2"
```

Comment:
You need to add the other packages you are using here:
* `@jupyterlab/docregistry`
* `@lumino/commands`
* `@lumino/apputils`
* `@lumino/coreutils`

And you can remove those you don't use `@juptyerlab/launcher` and `@jupyterlab/mainmenu`

Reply:
done